### PR TITLE
Update readme kv

### PR DESCRIPTION
**Pull request by AlexGhiondea**
/cc @jongio @Petermarcu @loarabia

### CODE REVIEW

File: sdk/keyvault/Microsoft.Azure.KeyVault/README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Microsoft Azure Keyvault SDK for .NET
 
+## New libraries available
+
+This library has been replaced by the following new Azure SDKs. You can read about the new Azure SDKs at https://aka.ms/azsdkvalueprop.
+
+The latest libraries to interact with the Azure KeyVault service are:
```
Comment (heaths) [inline sdk/keyvault/Microsoft.Azure.KeyVault/README.md:7]: "Key Vault" is the actual product name. We should use that more consistently in public documentation, at least.

File: sdk/keyvault/Microsoft.Azure.KeyVault/README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+* [Azure.Security.KeyVault.Keys](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Keys)
```
Comment (heaths) [inline sdk/keyvault/Microsoft.Azure.KeyVault/README.md:9]: Nit: I'd alphabetize the libraries as we do in other docs in track 2.

**Review by heaths — APPROVED** [review]

File: sdk/keyvault/Microsoft.Azure.KeyVault/README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 # Microsoft Azure Keyvault SDK for .NET
```
Comment (jongio) [inline sdk/keyvault/Microsoft.Azure.KeyVault/README.md:1]: Key Vault, not Keyvault

**Review by jongio — DISMISSED** [review]

File: sdk/keyvault/Microsoft.Azure.KeyVault/README.md (hunk 3 above)
Comment (AlexGhiondea) [reply sdk/keyvault/Microsoft.Azure.KeyVault/README.md:1]: Fixed.

File: sdk/keyvault/Microsoft.Azure.KeyVault/README.md [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+
+This library has been replaced by the following new Azure SDKs. You can read about the new Azure SDKs at https://aka.ms/azsdkvalueprop.
+
+The latest libraries to interact with the Azure Key Vault service are:
+
+* [Azure.Security.KeyVault.Certificates](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Certificates)
+* [Azure.Security.KeyVault.Keys](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Keys)
+* [Azure.Security.KeyVault.Secrets](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Secrets)
+
+It is **recommended** that you move to the new package.
```
Comment (heaths) [inline sdk/keyvault/Microsoft.Azure.KeyVault/README.md:13]: More than one. Maybe "...move to one or more of the new packages."

**Review by heaths — APPROVED** [review]

**Review by heaths — APPROVED** [review]

**Pull request merged**